Marlin/src/libs/MAX31865.cpp

```diff
@@ -484,17 +457,34 @@ uint8_t MAX31865::spixfer(uint8_t x) {
   if (_sclk == TERN(LARGE_PINMAP, -1UL, -1))
     return SPI.transfer(x);
 
-  uint8_t reply = 0;
-  for (int i = 7; i >= 0; i--) {
-    reply <<= 1;
-    WRITE(_sclk, HIGH);
-    WRITE(_mosi, x & (1 << i));
-    WRITE(_sclk, LOW);
-    if (READ(_miso))
-      reply |= 1;
-  }
+  #ifdef TARGET_LPC1768
+    return swSpiTransfer(x, _spi_speed, _sclk, _miso, _mosi);
+  #else
+    uint8_t reply = 0;
+    for (int i = 7; i >= 0; i--) {
+      WRITE(_sclk, HIGH);           DELAY_NS_VAR(_spi_delay);
+      reply <<= 1;
+      WRITE(_mosi, x & _BV(i));     DELAY_NS_VAR(_spi_delay);
+      if (READ(_miso)) reply |= 1;
+      WRITE(_sclk, LOW);            DELAY_NS_VAR(_spi_delay);
+    }
+    return reply;
+  #endif
+}
 
-  return reply;
+void MAX31865::softSpiBegin(const uint8_t spi_speed) {
+  #ifdef MAX31865_DEBUG
+    SERIAL_ECHOLNPGM("Initializing MAX31865 Software SPI");
+  #endif
+  #ifdef TARGET_LPC1768
+    swSpiBegin(_sclk, _miso, _mosi);
+    _spi_speed = swSpiInit(spi_speed, _sclk, _mosi);
+  #else
+    _spi_delay = (100UL << spi_speed) / 3; // Calculate delay in ns. Top speed is ~10MHz, or 100ns delay between bits.
```

Comment:
Not sure if 10MHz is okay for MAX31865. [Datasheet](https://datasheets.maximintegrated.com/en/ds/MAX31865.pdf) mentions a max clock frequency of 5MHz

Reply:
Accordingly, the `SPI_QUARTER_SPEED` used here should be limiting the speed to 2.5MHz.